examples/bulk/transport.ts

```diff
@@ -0,0 +1,60 @@
+import { readFileSync } from "fs";
+import Nodemailer from "nodemailer";
+import { MailtrapTransport } from "mailtrap"
+
+/**
+ * For this example, you need to have ready-to-use sending domain or,
+ * a Demo domain that allows sending emails to your own account email.
+ * @see https://help.mailtrap.io/article/69-sending-domain-setup
+ */
+
+const TOKEN = "<YOUR-TOKEN-HERE>"
+const SENDER_EMAIL = "<SENDER@YOURDOMAIN.COM>";
+const RECIPIENT_EMAIL = "<RECIPIENT@EMAIL.COM>";
```

Comment:
IMO emails should be lowercased, indeed email addresses are automatically lowercased by the protocol, so uppercase addresses make little sense.

Reply:
@leonid-shevtsov it's just a placeholder to hint a user to write his/her own

Reply:
right! I guess it makes sense 👍